### PR TITLE
installation docs edits

### PR DESCRIPTION
**Pull request by ryyeung**
installation docs edits

### CODE REVIEW

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 1]
```diff
@@ -33,6 +33,10 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:37]: Can we link the wallet creation docs here.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 2]
```diff
@@ -18,12 +18,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
 On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linux/download-install-and-setup-the-linux-development-workload?view=msvc-170#linux-setup-ubuntu-on-wsl).
 :::
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:22]: Can we link the wallet creation docs here.

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 3]
```diff
@@ -33,6 +33,10 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx:37]: Can we link the wallet creation docs here.

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 4]
```diff
@@ -18,12 +18,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
 On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linux/download-install-and-setup-the-linux-development-workload?view=msvc-170#linux-setup-ubuntu-on-wsl).
 :::
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:22]: Can we link the wallet creation docs here.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 5]
```diff
@@ -18,12 +18,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+:::
+
 ## Step 1: Download the Space Acres Installer
 
 <DownloadButtons />
 
 ## Step 2: Install Space Acres
 
+:::caution
+Depending on your system, your screen may look different than the screenshots below.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:32]: Can we get some macOS screenshots of the installation process and the Space Acres UI. Two different tabbed sections would be ideal.

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 6]
```diff
@@ -18,12 +18,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+:::
+
 ## Step 1: Download the Space Acres Installer
 
 <DownloadButtons />
 
 ## Step 2: Install Space Acres
 
+:::caution
+Depending on your system, your screen may look different than the screenshots below.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:32]: Can we get some macOS screenshots of the installation process and the Space Acres UI. Two different tabbed sections would be ideal.

**Review by jim-counter — CHANGES_REQUESTED** [review]

**Comment by ryyeung** [comment]
@jim-counter Addressed comments, awaiting review.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 7]
```diff
@@ -44,6 +54,18 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
 
     ![space-acres-install-5](/img/doc-imgs/space-acres/space-acres-install-5.png)
 
+</TabItem>
+
+<TabItem value="macos" label="🍎macOS" default>
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:59]:
```suggestion
<TabItem value="macos" label="🍎macOS">
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 8]
```diff
@@ -44,6 +54,18 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
 
     ![space-acres-install-5](/img/doc-imgs/space-acres/space-acres-install-5.png)
 
+</TabItem>
+
+<TabItem value="macos" label="🍎macOS" default>
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:59]:
```suggestion
<TabItem value="macos" label="🍎macOS">
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 9]
```diff
@@ -18,12 +20,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+:::
+
 ## Step 1: Download the Space Acres Installer
 
 <DownloadButtons />
 
 ## Step 2: Install Space Acres
 
+<Tabs groupId="Space_Acres_Installation">
+<TabItem value="otheros" label="Other OS" default>
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:35]:
```suggestion
<TabItem value="windows" label="Windows" default>
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 10]
```diff
@@ -18,12 +20,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+:::
+
 ## Step 1: Download the Space Acres Installer
 
 <DownloadButtons />
 
 ## Step 2: Install Space Acres
 
+<Tabs groupId="Space_Acres_Installation">
+<TabItem value="otheros" label="Other OS" default>
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:35]:
```suggestion
<TabItem value="windows" label="Windows" default>
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Can we add a Ubuntu tab too? You could use VirtualBox or similar to spin up a Linux VM for the screenshots.

Some guidance on paths is also a hot topic - can we make a suggestion on where to store node & farm data?

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/platforms/_download.mdx [hunk 11]
```diff
@@ -4,28 +4,28 @@ import styles from '@site/src/pages/index.module.css';
 <div className={`${styles.buttons} ${styles.flexContainer}`}>
   <Link
     className="button button--secondary"
-    to="https://github.com/subspace/space-acres/releases/download/0.1.28/space-acres-0.1.28-x86_64.msi"
+    to="https://github.com/subspace/space-acres/releases/download/0.1.29/space-acres-0.1.29-x86_64.msi"
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/platforms/_download.mdx:7]: Unrelated

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Almost there, Ryan!

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 12]
```diff
@@ -18,11 +23,64 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+        style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', textAlign: 'center', padding: '10px' }}>
+        <span style={{ fontSize: '20px' }}>Linux Portable App</span>
+        <span style={{ fontSize: '14px' }}>(AppImage)</span>
+    </Link>
+</div>
+
+# Step 2: Install Space Acres
+
+1. Open the downloaded Space Acres App Image.
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-1.png)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:58]: We need to add an empty line between the previous line and this line

Comment (EmilFattakhov) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:58]: Screenshot 2024-08-06 at 4 20 17 PM

Otherwise, this image is aligned incorrectly. This goes for all images

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 13]
```diff
@@ -18,11 +23,64 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+        <span style={{ fontSize: '14px' }}>(AppImage)</span>
+    </Link>
+</div>
+
+# Step 2: Install Space Acres
+
+1. Open the downloaded Space Acres App Image.
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-1.png) 
+2. Mark the AppImage as "Executable as Program".
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-2.png)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:60]: An empty line above here too

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 14]
```diff
@@ -44,9 +102,36 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
...
+        <span style={{ fontSize: '20px' }}>macOS Installer</span>
+        <span style={{ fontSize: '14px' }}>(.dmg)</span>
+    </Link>
+</div>
+
+# Step 2: Install Space Acres
+
+1. Open the downloaded Space Acres installer. Drag Space Acres into your Applications folder.
+    
+    ![](/img/doc-imgs/space-acres/mac-space-acres-setup-1.png)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:126]: This one is done correctly, so the spacing is correct

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 15]
```diff
@@ -44,9 +102,36 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
...
+    ![](/img/doc-imgs/space-acres/mac-space-acres-setup-1.png) 
+2. Double-click the Space Acres application to open!
+
+</TabItem>
+</Tabs>
+
 ## Step 3: Initalize Space Acres
 
-1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](../prerequisites.mdx)  
+1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](docs/farming-&-staking/farming/prerequisites.mdx)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:134]: Better to move the dot out of the link

```suggestion
1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide](docs/farming-&-staking/farming/prerequisites.mdx).
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 16]
```diff
@@ -66,7 +151,7 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
     - Autonomys DSN P2P Port
     - Enable Faster Networking
 
-    For more information on networking and its implications, refer to our [networking guide.](../additional-guides/network-info.md)
+    For more information on networking and its implications, refer to our [networking guide.](docs/farming-&-staking/farming/additional-guides/network-info.md)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:154]:
```suggestion
    For more information on networking and its implications, refer to our [networking guide](docs/farming-&-staking/farming/additional-guides/network-info.md).
```

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 17]
```diff
@@ -33,6 +33,11 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js.](../../wallets/polkadot.md)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx:38]:
```suggestion
To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js.](../../wallets/polkadot.md).
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 18]
```diff
@@ -18,11 +23,63 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
 On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linux/download-install-and-setup-the-linux-development-workload?view=msvc-170#linux-setup-ubuntu-on-wsl).
 :::
 
-## Step 1: Download the Space Acres Installer
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js.](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:28]:
```suggestion
To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
```

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 19]
```diff
@@ -33,6 +33,11 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js.](../../wallets/polkadot.md)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:38]:
```suggestion
To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js](../../wallets/polkadot.md).
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 20]
```diff
@@ -18,11 +23,63 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+        style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', textAlign: 'center', padding: '10px' }}>
+        <span style={{ fontSize: '20px' }}>Linux Portable App</span>
+        <span style={{ fontSize: '14px' }}>(AppImage)</span>
+    </Link>
+</div>
+
+# Step 2: Install Space Acres
+
+1. Open the downloaded Space Acres App Image.
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-1.png)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:58]: Spacing

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 21]
```diff
@@ -18,11 +23,63 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+        <span style={{ fontSize: '14px' }}>(AppImage)</span>
+    </Link>
+</div>
+
+# Step 2: Install Space Acres
+
+1. Open the downloaded Space Acres App Image.
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-1.png) 
+2. Mark the AppImage as "Executable as Program".
+    ![](/img/doc-imgs/space-acres/linux-space-acres-setup-2.png)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:60]: Spacing

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 22]
```diff
@@ -66,7 +150,7 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
     - Autonomys DSN P2P Port
     - Enable Faster Networking
 
-    For more information on networking and its implications, refer to our [networking guide.](../additional-guides/network-info.md)
+    For more information on networking and its implications, refer to our [networking guide.](versioned_docs/version-latest/farming-&-staking/farming/additional-guides/network-info.md)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:153]:
```suggestion
    For more information on networking and its implications, refer to our [networking guide](versioned_docs/version-latest/farming-&-staking/farming/additional-guides/network-info.md).
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 23]
```diff
@@ -44,9 +101,36 @@ On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linu
...
+    ![](/img/doc-imgs/space-acres/mac-space-acres-setup-1.png) 
+2. Double-click the Space Acres application to open!
+
+</TabItem>
+</Tabs>
+
 ## Step 3: Initalize Space Acres
 
-1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](../prerequisites.mdx)  
+1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](versioned_docs/version-latest/farming-&-staking/farming/prerequisites.mdx)
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:133]:
```suggestion
1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide](versioned_docs/version-latest/farming-&-staking/farming/prerequisites.mdx).
```

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Just a few small things left

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 24]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Install
 sidebar_position: 2
-description: How to run an Autonomys Network Farmer with the Substrate CLI
+description: How to run a Subspace Network Farmer with the Substrate CLI
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:4]: Unrelated and inaccurate

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 25]
```diff
@@ -20,7 +20,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 :::tip 
-[Releases](https://github.com/autonomys/subspace/releases) section on our GitHub is the best place to get fresh releases of binaries and check out the source code.
+[Releases](https://github.com/subspace/subspace/releases) section on our GitHub is the best place to get fresh releases of binaries and check out the source code.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:23]: Unrelated and inaccurate

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 26]
```diff
@@ -79,9 +79,9 @@ Your insights on our releases elevate our progress. Fill out this [form](https:/
 
 :::note Learn More About Syncing, Plotting & Farming
 
-If you would like to learn more about how the syncing, plotting and farming processes works on the Autonomys Network, please refer to our Autonomys Academy links below. <br />
-- [Synchronization](https://academy.autonomys.xyz/subspace-protocol/network-architecture/networking-protocols#synchronization) <br />
-- [Plotting](https://academy.autonomys.xyz/subspace-protocol/consensus/proof-of-archival-storage/plotting) <br />
-- [Farming](https://academy.autonomys.xyz/subspace-protocol/consensus/proof-of-archival-storage/farming) <br />
+If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Autonomys Academy links below. <br />
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:82]: All three next sentences shouldn't be changed

File: versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 27]
```diff
@@ -35,7 +35,7 @@ There are multiple versions of our releases available below.
 
 :::note
 Please note that a wallet with a Substrate address is required to run the software.<br />
-To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js.](../../wallets/polkadot.md)
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx:38]: This latest version is correct, let's just copy this to pre-release until Randy removes it

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 28]
```diff
@@ -33,8 +33,13 @@ There are multiple versions of our releases available below.
...
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
+:::
+
 We will download two files for your respective operating system.
-1. `Subspace-Node` - This is the executable that actually makes the connection with the Autonomys network
+1. `Subspace-Node` - This is the executable that actually makes the connection with the Subspace Network
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:42]: This one is also left out

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 29]
```diff
@@ -33,8 +33,13 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:38]: This is not critical since we're likely removing the pre-release docs, but the links should lead to just /docs instead of /versioned_docs. They were correct previously.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 30]
```diff
@@ -18,11 +23,68 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
...
+    <Link
+        className="button button--secondary"
+        to="https://github.com/subspace/space-acres/releases/download/0.1.33/space-acres_0.1.33-1_amd64.deb"
+        style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center', textAlign: 'center', padding: '10px' }}>
+        <span style={{ fontSize: '20px' }}>Linux Installer</span>
+        <span style={{ fontSize: '14px' }}>(.DEB)</span>
+        </Link>
+        <Link
+        className="button button--secondary"
+        to="https://github.com/subspace/space-acres/releases/download/0.1.33/space-acres-0.1.33-x86_64.AppImage"
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:48]: Sorry I missed that before - we should adjust all links from _https://github.com/subspace/space-acres_ to _https://github.com/autonomys/space-acres_
